Babylon/Mesh/babylon.mesh.ts

```diff
@@ -904,17 +904,32 @@
             if (!this.isVerticesDataPresent(VertexBuffer.NormalKind)) {
                 return;
             }
-
             data = this.getVerticesData(VertexBuffer.NormalKind);
             temp = [];
+            var temp_normal = new Vector3(0, 0, 0);    // this vector is used to normalize the newly transformed normal
             for (index = 0; index < data.length; index += 3) {
                 Vector3.TransformNormal(Vector3.FromArray(data, index), transform).toArray(temp, index);
```

Comment:
Normalizing does make sense! :-)
Doesn't this function already deliver a normal? You can simply add the normalize function between the TransformNormal and the toArray. Or did I miss something?